src/db/moderator.ts

```diff
@@ -0,0 +1,49 @@
+import { Snowflake } from 'discord.js';
+import { Schema } from 'mongoose';
+
+/**
+ * A moderator is similar to a user, except
+ * we **do not** keep track of previous usernames
+ * or nicknames.
+ *
+ * This isn't saved by itself, rather it is used to "log"
+ * actions taken by an admin.
+ */
+export interface Moderator {
+  /**
+   * The discord server userId for the mod
+   */
+  userId: string | Snowflake;
+  /**
+   * The discord server nickname of the mod
+   */
```

Comment:
On this one, I believe the `user` value is a Snowflake, but storing the `user.id` value is a string. We should type it as only one or the other to enforce consistency in the codebase, right?

Reply:
I might of left this `Snowflake` in here on accident.